met-web/src/components/common/Table/index.tsx

```diff
@@ -90,6 +91,7 @@ function MetTable<T>({
     pageInfo = {
         total: rows.length,
     },
+    emptyText = 'No data to display',
```

Comment:
Minor thing to consider is a more non tech user friendly text like "No records were found", maybe

Reply:
Done, thanks